Makefile

```
@@ -0,0 +1,44 @@
+.PHONY: docs docs-components docs-test docs-verify fmt fmt-check lint test hooks release-docs
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)